docs/configuration/index.md

```diff
@@ -983,6 +983,37 @@ useful if you want work evenly distributed across your MiddleManagers.
 |`type`|`equalDistribution`.|required; must be `equalDistribution`|
 |`affinityConfig`|[Affinity config](#affinity) object|null (no affinity)|
 
+###### Equal Distribution With Category Spec
+
+This strategy is a variant of `Equal Distribution`, which support `workerCategorySpec` field rather than `affinityConfig`. By specifying `workerCategorySpec`, you can assign tasks to run on different categories of MiddleManagers based on the tasks' **taskType** and **dataSource name**. This strategy can't work with `AutoScaler` since the behavior is undefined.
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`type`|`equalDistributionWithCategorySpec`.|required; must be `equalDistributionWithCategorySpec`|
+|`workerCategorySpec`|[Worker Category Spec](#WorkerCategorySpec) object|null (no worker category spec)|
```

Comment:
it looks like these anchor links should be all lower case, I think that should fix CI